scripts/logbook.py

```diff
@@ -0,0 +1,245 @@
+
+from genologics.entities import *
+from genologics.lims import *
+from genologics.config import BASEURI, USERNAME, PASSWORD
+from genologics.descriptors import StringDescriptor,EntityDescriptor
+
+from __future__ import print_function
+import httplib2
+import os
+
+from apiclient import discovery
+from oauth2client import client
+from oauth2client import tools
+from oauth2client.file import Storage
+
+try:
+    import argparse
+    flags = argparse.ArgumentParser(parents=[tools.argparser]).parse_args()
+except ImportError:
```

Comment:
Any particular reason for this `try` ? I thought `argparse` is one of builtin module of python, even though if not if necessary it could be installed under the `env`

Reply:
No particular reason for this `try` actually. Just used the example codes from Google. I will remove it.